examples/Atlas/+atlasParams/Walking.m

```diff
@@ -0,0 +1,26 @@
+classdef Walking < atlasParams.Base
+  methods
+    function obj = Walking(r)
+      typecheck(r, 'Atlas');
+      obj = obj@atlasParams.Base(r);
+      obj.whole_body.w_qdd = zeros(r.getNumVelocities(), 1);
+      obj.whole_body.w_qdd(findPositionIndices(r, 'arm')) = .0001;
+      obj.whole_body.w_qdd(findPositionIndices(r, 'back')) = .0001;
+      obj.whole_body.w_qdd(r.findPositionIndices('back_bkx')) = 0.1;
+      obj.whole_body.damping_ratio = 0.5;
+      obj.body_motion(r.findLinkId('pelvis')).Kp = [nan; nan; 20; 20; 20; 20];
+      obj.body_motion(r.findLinkId('pelvis')).damping_ratio = 0.5;
+      obj.body_motion(r.foot_body_id.right).Kp = [100; 100; 100; 150; 150; 150];
+      obj.body_motion(r.foot_body_id.right).damping_ratio = 0.5;
+      obj.body_motion(r.foot_body_id.left).Kp = [100; 100; 100; 150; 150; 150];
+      obj.body_motion(r.foot_body_id.left).damping_ratio = 0.5;
+      [obj.body_motion.weight] = deal(0.15);
+      obj.body_motion(r.findLinkId('pelvis')).weight = 0.075;
```

Comment:
Gotcha. I'm going to implement a WalkingHardware (or maybe a different name?) param set in DRC which derives from Walking and sets all of our hardware-specific gains. I'll make sure these gains get copied over to that set.